mlflow/server/js/src/components/ExperimentViewUtil.js

```diff
@@ -83,6 +83,14 @@ export default class ExperimentViewUtil {
     ];
   }
 
+  static getSortValue({metricsList, paramsList, tagsList, idx, sortState, runInfo}) {
+    return ExperimentViewUtil.computeSortValue(sortState,
```

Comment:
Why do we need `computeSortValue` and `getSortValue`?

Reply:
Analogously why do we need to continue calculating the sortValue in `this.getRow`?

Reply:
Good question, I don't think we need both methods or to compute the sort value in getRow, will fix that